rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -268,6 +269,9 @@ const auto spu_putllc_tx = build_function_asm<bool(*)(u32 raddr, u64 rtime, cons
 	build_transaction_abort(c, 0xff);
 	c.xor_(x86::eax, x86::eax);
 	c.ret();
+	c.bind(retry);
+	c.mov(x86::eax, 2);
```

Comment:
no transaction exit on retry? Is RTM safely re-entrant?

Reply:
good catch

Reply:
if it got here it already aborted internally due to a memory conflict, xabort will just be a nop here.

Reply:
I've corrected last issue with putllc abort fix, where comparison fail while being the last on timeout led to the execution of useless retry (by moving timeout check after fail check).
If there are more issues in this pr please let me know, otherwise its mergeable imo.